F1 Standings: Allow changing season.

sxn opened the pull request:
## Description

Updates `@raycast/api` to the latest version and makes use of the new [List.Dropdown](https://developers.raycast.com/api-reference/user-interface/list#list.dropdown) to allow changing seasons.

## Screencast

https://user-images.githubusercontent.com/1155589/158073090-51f0902b-5c08-4bca-ae37-9571fbf2ce36.mov

https://user-images.githubusercontent.com/1155589/158073106-68ad6013-4bbc-4f10-a003-f1c31b9e3c9e.mov

https://user-images.githubusercontent.com/1155589/158073107-f91d09bf-b689-4f3e-8a74-d22c0c6970a7.mov

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed in the `media` folder

csigritz commented on extensions/f1-standings/src/useSeasons.ts:
```diff
@@ -0,0 +1,34 @@
+  const [seasons, setSeasons] = useState<Season[]>([]);
+
+  useEffect(() => {
+    async function fetchDrivers() {
```
Maybe this should be called "fetchSeasons"

sxn replied:
yup!

csigritz reviewed (changes requested):
Thx! Did not know that the top dropdown is now available for community extensions. Only please rename the function in the useSeassons file

csigritz commented:
Testet it out. Makes the extensions a lot more useful!

sxn commented:
> Thx! Did not know that the top dropdown is now available for community extensions. Only please rename the function in the useSeassons file

it was added [a couple of days ago](https://developers.raycast.com/changelog#1.30.0-2022-03-09)! 

I've renamed the function (good catch, thanks), as well as fixed an issue I introduced w/ the constructors command. Apparently - the API doesn't return constructor standings data for 1950-1957, so the command crashed 😬 

I've also added driver & constructor country flags based on the `nationality` field returned by the API. Let me know if you don't want it, or if you want to change how it looks etc.

<img width="500" alt="image" src="https://user-images.githubusercontent.com/1155589/158075714-af3d6b92-771c-41d2-8b38-92426faeef88.png">
<img width="500" alt="image" src="https://user-images.githubusercontent.com/1155589/158075715-0501c94c-5cc7-425d-9e94-30ec52b34f92.png">

csigritz commented:
The flags are great! Could you add them also to the Race Results detail view in the Race Schedule command?

sxn commented:
yup, was just working on that 😺

csigritz reviewed (approved):
🎉

sxn commented:
updated the PR so that flags are shown for races, as well as on results pages:

<img width="500" alt="image" src="https://user-images.githubusercontent.com/1155589/158078324-a1ed3d3f-0e97-400d-b6aa-f002b2c8f514.png">
<img width="500" alt="image" src="https://user-images.githubusercontent.com/1155589/158078340-76d35f63-7c36-4566-9a67-de4d0b8078d3.png">
